StereoKitC/stereokit.h

```diff
@@ -841,7 +841,7 @@ SK_API tex_t        tex_create_cubemap_files(const char **cube_face_file_xxyyzz,
 SK_API void         tex_set_id              (tex_t texture, const char *id);
 SK_API const char*  tex_get_id              (const tex_t texture);
 SK_API void         tex_set_fallback        (tex_t texture, tex_t fallback);
-SK_API void         tex_set_surface         (tex_t texture, void *native_surface, tex_type_ type, int64_t native_fmt, int32_t width, int32_t height, int32_t surface_count);
+SK_API void         tex_set_surface         (tex_t texture, void *native_surface, bool32_t owned, tex_type_ type, int64_t native_fmt, int32_t width, int32_t height, int32_t surface_count);
```

Comment:
For public API changes like this, `bool32_t owned` should be the last item in the signature, and should have an `sk_default(true)` that matches the original behavior.

Reply:
This should mean that most of the rest of the changes can be dropped

Reply:
do you still want to expose that to C#?

Reply:
Yeah! C# similarly should have a default value for owned, and I generally prefer for the signatures to be in the same order as the C function.